pyquil/external/rpcq.py

```diff
@@ -1,56 +1,163 @@
+import json
 from typing import Dict, List, Union, Optional, Any
-from typing_extensions import Literal
-from pydantic import BaseModel, Field
 from rpcq.messages import TargetDevice as TargetQuantumProcessor
+from dataclasses import dataclass, field
 
+#JsonValue = Union[type(None), bool, int, float, str, list["JsonValue"], dict[str, "JsonValue"]]
+JsonValue = Any
```

Comment:
We'll have to fix this. Why did your commented out version not work? Something like this should be doable. Maybe we have to update `mypy`? I think support for recursive type definitions is new-ish.

```py
JSON: TypeAlias = Union[dict[str, "JSON"], list["JSON"], str, int, float, bool, None]
```

Reply:
I think upgrading mypy fixed this